src/timer.py

```diff
@@ -1,5 +1,5 @@
 import pygame
-
+from .settings import *
```

Comment:
What exactly do you use from settings here? No need to import anything from there, is there?

Reply:
After I created this PR, when I synced my code, all the headers changed and broke a lot of the code. I went back through and just added the wildcard headers everywhere they had been before to fix the errors. I may have accidentally imported them here thinking it was required.